surf/apps/communities/views.py

```diff
@@ -45,19 +46,9 @@ class CommunityViewSet(ListModelMixin,
     serializer_class = CommunitySerializer
     permission_classes = []
 
-    def get_serializer_class(self):
-        """
-        Returns serializer class depending on action method
-        """
-
-        if self.action == 'update':
-            return CommunityUpdateSerializer
-
-        return CommunitySerializer
-
     def update(self, request, *args, **kwargs):
         # only active admins can update community
-        self._check_access(request.user, instance=self.get_object())
+        #check_access_to_community(request.user, instance=self.get_object())
```

Comment:
What is this?

Reply:
Me forgetting to uncomment a line for testing 😅